spec/system/admin/products_v3/products_spec.rb

```diff
@@ -1446,6 +1446,34 @@
 
       expect(page).to have_selector row_containing_name(product_supplied_inactive.name)
     end
+
+    it "allows me to create a product" do
```

Comment:
I thought this was already covered by "creating a new product" on line 626?

Reply:
Yes, you're right. I've added it anyway because the context was on enterprise permissions. But indeed it's doing the same as the test on line 626 (I've opted to only test the redirect here) and - more thoroughly - the test on `./spec/system/admin/products_spec.rb` ([here](https://github.com/openfoodfoundation/openfoodnetwork/blob/e22bec014bd0318131dba17461b18a46bde0ab0b/spec/system/admin/products_spec.rb#L29)). Removing it :+1: